src/swap/sideshift.js

```diff
@@ -235,13 +242,17 @@ const createFetchSwapQuote = (api: SideshiftApi, affiliateId: string) =>
 
     const isEstimate = false
 
+    const destinationTag = order.depositAddress.destinationTag
```

Comment:
FlowJS calls it [sketchy-null](https://github.com/facebook/flow/blob/main/website/en/docs/linting/rule-reference.md#sketchy-null-)